### ISSUE TITLE
Missing Message API endpoints

### ISSUE THREAD

**Issue by Foorack** [vrchatapi/specification#33]
### `https://vrchat.com/api/1/message/<userId>/message`
```javascript
[
	{
		"id":"default",
		"slot":0,
		"message":"Come hang out with me!",
		"messageType":"message",
		"updatedAt":"2017-02-01T21:59:00.000+0000",
		"remainingCooldownMinutes":0,
		"canBeUpdated":true
	},
	{
		"id":"default",
		"slot":1,
		"message":"Come hang out with everyone!",
		"messageType":"message",
		"updatedAt":"2017-02-01T21:59:00.000+0000",
		"remainingCooldownMinutes":0,
		"canBeUpdated":true
	},
       ...
]
```

Trying to view someone else's messages other than your own gives `You are not authorized to perform that action․` error.

### `https://vrchat.com/api/1/message/<userId>/message/<slot>`

```json
{
	"id":"default",
	"slot":0,
	"message":"Come hang out with me!",
	"messageType":"message",
	"updatedAt":"2017-02-01T21:59:00.000+0000",
	"remainingCooldownMinutes":0,
	"canBeUpdated":true
}
```

There's also surely PUT requests to update these messages, etc. Also document how they are used with invites.

**Comment by Foorack** [comment]
Updating a message changes it's "ID" from default to a real ID:

```javascript

    {
        "id": "invm_24a1c14d-5e24-48e5-90e3-c3f712420ffa",
        "slot": 11,
        "message": "Sorry I can't talk in VR right nowǃ Can I add you on Discord？",
        "messageType": "message",
        "updatedAt": "2021-03-12T20:45:01.355Z",
        "remainingCooldownMinutes": 0,
        "canBeUpdated": true
    }
```

Now we have `invm_` as ID-prefix as well.

**Comment by TomAvecLeVdsl** [comment]
When used in an invite, they apear as `"inviteMessage" : "Come hang out with me!"` in the details part :) if that can help 🙂

**Comment by Foorack** [comment]
> When used in an invite, they apear as `"inviteMessage" : "Come hang out with me!"` in the details part :)

I guess that's part of the Notification object sent by #37 and not part of the InviteMessage block. This issue specifically is just how to edit the pre-written messages. But thanks!

>  if that can help :slightly_smiling_face: 

Yes! **Any** information at all helps a lot in documenting. It saves me from having to discover everything, and instead allows me to simply *verify* and then document.

**Comment by TomAvecLeVdsl** [comment]
How to edit an invite message : 
send a PUT request to "https://api.vrchat.cloud/api/1/message/usr_your-awsome-user-id/message/5" with your api key and auth cookie of corse, and send in the body `
{
    "message": "This is the edited message"
}`
Hope this helps

**Comment by TomAvecLeVdsl** [comment]
After editing a message, we seems to have a 60 minutes countdown before we can edit it again
